docs/guide/getting-started.md

```diff
@@ -0,0 +1,171 @@
+# Getting started
```

Comment:
This page is 99% copied from the README. Only change is in the first paragraph because I removed the first line. Will make a proper "getting started" page another time.